sublime_linter.py

```diff
@@ -332,22 +316,25 @@ def join_msgs(cls, line_dict, we_count, show_count=False):
                 <div>{messages}</div>
             '''
 
-        template = "{linter}: {code} - {escaped_msg}"
-        template_no_code = "{linter}: {escaped_msg}"
+        tmpl_with_code = "{linter}: {code} - {escaped_msg}"
+        tmpl_sans_code = "{linter}: {escaped_msg}"
 
```

Comment:
I also did not find a better way to conditionally include ` - ` based upon noneness of `code`.
AFAIK the string formatting mini lang does not support conditions.

Reply:
None of Python's default templating engines support this functionality. A more advanced template engine like `jinja2` would be necessary for this, but this just seems overkill here.

Reply:
Alternatively, `sublime.expand_variables` could be abused with replacements on the `code` variable for the `" - "` segment, though that would make things less clear imo.

(i.e. `${code}${code/.+/ - }`, assuming an empty `code` is an empty string)

Reply:
The templates are only used in one place, and now they’re clearly named. I wouldn’t put more thought into it at this point.

Reply:
Thanks for the input guys. We'll just leave it the way it is.